### PR TITLE
Add guidance on using Corso repos

### PR DESCRIPTION
**Pull request by ntolia**
## Description

Clarify how Corso repositories and object storage buckets are related.

## Does this PR need a docs update or release note?

- [x] :no_entry: No 

## Type of change

- [x] :world_map: Documentation

### CODE REVIEW

File: website/docs/setup/repos.md [hunk 1]
```diff
@@ -10,9 +10,15 @@ import TabItem from '@theme/TabItem';
 import TOCInline from '@theme/TOCInline';
 import {Version} from '@site/src/corsoEnv';
 
-A Corso [repository](../concepts#corso-concepts) stores encrypted copies of your backup data. Corso uses
+A Corso [repository](../concepts#corso-concepts) stores encrypted copies of a Microst 365 tenant's
+backup data. Each repository is configured to store data in an object storage bucket and, optionally,
+a user-specified prefix within the bucket. A repository is only meant to store a single tenant's data
+but a single boject storage bucket can contain multiple repositories if unique `--prefix` options are
```
Comment (gmatev) [inline website/docs/setup/repos.md:16]:
```suggestion
but a single object storage bucket can contain multiple repositories if unique `--prefix` options are
```

**Review by gmatev — APPROVED** [review]

File: website/docs/setup/repos.md [hunk 2]
```diff
@@ -10,9 +10,15 @@ import TabItem from '@theme/TabItem';
 import TOCInline from '@theme/TOCInline';
 import {Version} from '@site/src/corsoEnv';
 
-A Corso [repository](../concepts#corso-concepts) stores encrypted copies of your backup data. Corso uses
+A Corso [repository](../concepts#corso-concepts) stores encrypted copies of a Microst 365 tenant's
```
Comment (gmatev) [inline website/docs/setup/repos.md:13]:
```suggestion
A Corso [repository](../concepts#corso-concepts) stores encrypted copies of a Microsot 365 tenant's
```

Comment (gmatev) [reply website/docs/setup/repos.md:13]: Typo

**Comment by ntolia** [comment]
@gmatev Thanks for fixing the typos. I was heading to bed and was going to fix it this morning but I see you got to it before I did.

**Pull request merged**